doc/_stdlib_gen/stdlib-content.jsonnet

```diff
@@ -681,6 +681,20 @@ local html = import 'html.libsonnet';
             },
           ],
         },
+        {
+          name: 'parseCsv',
+          params: ['str', 'delimiter=","'],
+          availableSince: 'upcoming',
+          description: |||
+            Parses a CSV string into JSON.
```

Comment:
Needs to mention how the first row is interpreted as a header row and must have unique column names, which then are used as fields in the JSON representation.

Reply:
Updated